Bottom: 36c2f569c4b8537fb0c0830bef0761725bb3197f
Top:    ca2b9515d48e23a0598b30189087ce8de9d7255c
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-11 14:37:47 +0000

TMP alterartions to build-all.sh


---

diff --git a/w32/build-all.sh b/w32/build-all.sh
index 04aec82..689d307 100755
--- a/w32/build-all.sh
+++ b/w32/build-all.sh
@@ -2,9 +2,6 @@ NEVERBUILD="
 "
 
 NOBUILD="
-"
-
-BUILD="
 jpeg
 zlib
 libpng
@@ -17,6 +14,9 @@ glib
 atk
 cairo
 pango
+"
+
+BUILD="
 gtk+
 gd
 pcb
